persistent/tests/test_persistence.py

```diff
@@ -68,7 +71,7 @@ def register(self, obj):
         jar._cache = self._makeCache(jar)
         return jar
 
-    def _makeBrokenJar(self):
+    def _makeBrokenJar(self, exc=NotImplementedError):
```

Comment:
OK. I was imagining that `NotImplementedError` might be caught and handled specially somewhere so I wanted to be sure to have something that would definitely propagate. But that is hypothetical, I admit.